cmd/snap/cmd_list_test.go

```diff
@@ -65,9 +75,10 @@ func (s *SnapSuite) TestList(c *check.C) {
 	rest, err := snap.Parser(snap.Client()).ParseArgs([]string{"list"})
 	c.Assert(err, check.IsNil)
 	c.Assert(rest, check.DeepEquals, []string{})
-	c.Check(s.Stdout(), check.Matches, `Name +Version +Rev +Tracking +Publisher +Notes
-foo +4.2 +17 +potatoes +bar +-
-`)
+	c.Check(s.Stdout(), check.Equals, `
+Name  Version  Rev  Tracking  Publisher  Notes
+foo   4.2      17   potatoes  bar        blocked
```

Comment:
(bkesheed) I wonder if we should prepend something like "health:blocked" or "health-blocked" or so to give more context?